WAS-585 Add partner page

rickclare opened the pull request.

rickclare commented:
Over to @Cruikshanks for PR review

Cruikshanks commented on .htmlhintrc:
```diff
@@ -8,11 +8,11 @@
   "spec-char-escape": false,
-  "id-unique": false,
-  "src-not-empty": false,
+  "id-unique": true,
```
Could I ask you to create a wiki page in WEBO for htmlhint rules and stick these in there? From then on we have a source of truth for what the rules should be across our projects.

rickclare replied:
I've added a Trello task for this https://trello.com/c/7eIyEjIe/77-create-a-wiki-page-in-webo-for-htmlhint-rules

Cruikshanks commented on app/controllers/digital_services_core/enrollments_controller.rb:
```diff
@@ -1,3 +1,4 @@
+# rubocop:disable Metrics/AbcSize
```
Was this exception placed at the head rather than at the problem method for a specific reason? It's just if we're adding these I'd rather they were tied to the problem methods, rather than allowing new or changed methods to automatically be ignored.

rickclare replied:
Good point. I'll move it to be relevant to the specfic method

Cruikshanks commented:
A couple of comments raised but only one needs responding to within the context of the PR. Also just to let you know I'm not going to attempt to squash this one because;

- Have started using `rebase` have found that branches which feature merges are complex to squash and the risk of loosing something is to high
- The moving of the token stuff and the updates to rubocop probably should be in their own commits and not squashed in with the PR

Back to you @rickclare for a response and then we can get this in.

rickclare commented:
@Cruikshanks Hopefully I've addressed your comments, so over to you to merge.

Noted that I should split tasks such as "Rubocop amend" into seperate PRs.
